### DISCUSSION TITLE
nebula-graphd.conf

### DISCUSSION THREAD

**Discussion by porscheme** [vesoft-inc/nebula#4657] [Q&A]
Nebula version: v3.2.0

Below is what we have in nebula-graphd.conf

Cluster is working fine but below is confusing, should we change meta_server_addrs to comma separated meta server addresses? 

```
########## networking ##########
# Comma separated Meta Server Addresses
--meta_server_addrs=127.0.0.1:9559
```

**Comment by QingZ11** [comment]
For multiple services, the configuration is as below

![image](https://user-images.githubusercontent.com/38887077/216508968-84e7f33d-22bd-4299-a0a2-cc862a307ee2.png)

> should we change meta_server_addrs to comma separated meta server addresses?

is this the way you want?
